website/members/api/v2/serializers/member.py

```diff
@@ -47,3 +54,41 @@ class Meta:
             "membership_type",
             "profile",
         )
+
+
+class MemberCurrentSerializer(MemberSerializer):
```

Comment:
I think MemberDetailSerializer would be better (as this serializes a Member in detail, not necessarily the current member)

Reply:
No this is always the current member, you will never get this in any other case. This information should never be accidentally exposed in any other part of the API.

Reply:
Well the serializer itself doesn't care to serialize other member objects. But it's fine by me